quickshell/Services/NiriService.qml

```diff
@@ -1191,6 +1191,10 @@ Singleton {
                 kdlContent += `    backdrop-color "${niriSettings.backdropColor}"\n`;
             }
 
+            if (output.mirror && output.mirror.length > 0) {
```

Comment:
Niri doesnt have built-in display mirroring, this should be removed and the toggle should be hidden if not `CompositorService.isHyprland`